karavan-builder/openshift/karavan-builder-pvc.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: karavan-m2-cache
+spec:
+  resources:
+    requests:
+      storage: 10Gi
+  volumeMode: Filesystem
+  accessModes:
+    - ReadWriteOnce
+---
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: karavan-jbang-cache
+spec:
+  resources:
+    requests:
+      storage: 10Gi
```

Comment:
I do not think jbang cache needs as much space as m2 because we only have a fixed set of JARs for jbang - eg the dynamic download is done by camel and not jbang so we do not use it cache - only to bootstrap camel-jbang itself and the basic of camel

Reply:
I fixed it for 1Gi
However, this is only for demo purposes. Later we'll create an operator for karavan where this be a configurable parameter.